.github/workflows/linux_sanitizers.yml

```diff
@@ -51,6 +51,7 @@ jobs:
       BUILD_DIR: /__w/openvino/openvino/openvino_build
       LSAN_IGNORE: /__w/openvino/openvino/openvino/tests/lsan/suppressions.txt
       ASAN_IGNORE: /__w/openvino/openvino/openvino/tests/asan/suppressions.supp
+      TBB_ENABLE_SANITIZERS: 'ON'
```

Comment:
does cmake or other code rely on this option?

Reply:
@ilya-lavrenov , please see the conversation in the referenced ticket. There is a runtime check in TBB which is supposed to help.